source/manual/github-unavailable.html.md

```diff
@@ -6,59 +6,67 @@ layout: manual_layout
 ```
 
-1. Log in and push the image to ECR:
+1. Log into ECR and push the image:
 
 ```
-gds aws govuk-production-poweruser aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin $ECR_REGISTRY
-docker push $ECR_REGISTRY/$REPO:$IMAGE_TAG
+gds aws govuk-production-poweruser aws ecr get-login-password --region eu-west-1 | docker login --username AWS --password-stdin $REGISTRY
+docker push $REGISTRY/$REPO:$IMAGE_TAG
 ```
 
-### Deploy the image to EKS using Argo
-
-1. Get the Admin Argo credentials by running `kubectl -n cluster-services get secret argocd-initial-admin-secret -oyaml |yq .data.password |base64 -d`
-1. Log into [Argo](https://argo.eks.integration.govuk.digital/)
-1. Search for the application you wish to deploy in the "Search applications..." search bar
-1. Select the application from the list below the search bar
-1. Select "APP DETAILS"
-1. Go to "DISABLE AUTO-SYNC" then select "ok" when prompted "Disable Auto-Sync?"
-1. Close the "APP DETAILS" menu, then select the three dots next to the Deploy object for the part of the application you'd like to redeploy. For example, `account-api-worker` would be for deploying the `account-api-workers`.
-1. Go to the "LIVE MANIFEST" and select "EDIT". Find the "image", then update it to the new image tag you'd like to deploy. (You can double-check you have the correct image name by doing a `git log` after [checking out the code from CodeCommit](/manual/howto-checkout-and-commit-to-codecommit.html)).
-1. Click "Save" and Argo will start the deployment, which should complete in under ten minutes.
+### Deploy the image to Kubernetes using Argo CD
+
+> ⚠️ This procedure will disable all automatic deployments not only in the
+> environment you are working on but also in higher environments. For example,
+> performing this procedure on integration will prevent all automatic
+> deployments in integration, staging and production.
+
+1. Fetch the password for the Argo CD `admin` user. This varies by environment.
+
+    ```sh
+    kubectl -n cluster-services get secret argocd-initial-admin-secret -oyaml |yq .data.password |base64 -d
+    ```
+
+1. Log into Argo CD (for example in [integration](https://argo.eks.integration.govuk.digital/)).
+1. Open the `app-config` application.
+1. Select the `App Details` button near the top of the page.
+1. Scroll down to the bottom of the page and press `Disable auto-sync`. Argo will prompt you before actually disabling auto-sync.
+1. Go back to the main `Applications` page (the first page you see when you log into Argo CD).
+1. Go into the details page for the application you wish to deploy, either by selecting its card from the main page or using the `Search applications...` box and selecting it from the drop-down list.
+1. Select the `App details` button near the top of the page.
+1. Scroll down to the bottom of the page and press `Disable auto-sync`. Argo will prompt you before actually disabling auto-sync.
```

Comment:
This took me a re-read or two. I think it can be shortened and some of the steps re-used:

```suggestion
1. Disable the auto-sync for the `app-config` application by:
  1. Clicking the `Applications` link on the sidebar. then opening the `app-config` application (you can use the search bar at the top of the page if you can't see it)
  1. Select the `App Details` button near the top of the page.
  1. Scroll down to the bottom of the page and press `Disable auto-sync`. Argo will prompt you before actually disabling auto-sync.
1. Repeat the steps above for the application you wish to deploy (e.g. swap `app-config` for `collections-publisher` in the steps above).
```

Reply:
Good idea — I usually try to avoid loops in playbooks but I think this is a good exception. Makes the intent a bit clearer in this case I think. Thanks 👍